lib/components/Button/Button.js

```diff
@@ -10,13 +10,17 @@ const VARIANT = {
   NO_OUTLINE: 'NO_OUTLINE',
 };
 
+const VARIANT_DEFAULT = VARIANT.PRIMARY;
```

Comment:
I think we should discuss what the right 'default' should be -- I'm in favor of the "secondary" variant since it seems to be the more standard button type. My thinking is that the primary type is only used for submit buttons and the like, while secondary fills most all other uses

Reply:
That sounds good.

Reply:
@NathanAB do you have an opinion?

Reply:
I'm leaning towards primary but either way is fine